jiup/rules/rules.go

```diff
@@ -565,11 +565,11 @@ func init() {
 	Rule("geforce-experience",
 		v.Regexp(
 			"https://www.nvidia.com/en-us/geforce/geforce-experience/",
-			h.Re("https://us.download.nvidia.com/GFE/GFEClient/([0-9.]+)/GeForce_Experience"),
+			h.Re("https://.*.download.nvidia.com/GFE/GFEClient/([0-9.]+)/GeForce_Experience"),
```

Comment:
Can you change this to a `v.HTML` extractor?

Reply:
I suppose you want something similar to the download link extractor. Changed that.